plugins/filter_log_to_metrics/log_to_metrics.c

```diff
@@ -450,6 +450,35 @@ static int fill_labels(struct log_to_metrics_ctx *ctx, char **label_values,
     return label_counter;
 }
 
+/* Timer callback to inject metrics into the pipeline */
+static void cb_send_metric_chunk(struct flb_config *config, void *data)
+{
+    int ret;
+    struct log_to_metrics_ctx *ctx = data;
+
+    /* Check that metric context is not empty */
+    if (ctx->cmt == NULL || ctx->input_ins == NULL) {
+        return;
+    }
+    
+    if (ctx->new_data == FLB_TRUE) {
```

Comment:
Could you please ensure that you either use this type of comparison or the implicit `if (x) {` uniformly across the code?